src/AppInstallerCLI.sln

```diff
@@ -147,6 +147,17 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WindowsPackageManager", "Wi
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "COMServer", "COMServer\COMServer.vcxitems", "{409CD681-22A4-469D-88AE-CB5E4836E07A}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "manifests", "manifests", "{6FF8E5FE-C2D3-4991-8519-6FD12A457DDE}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "v1.2.0", "v1.2.0", "{6FA346E3-2171-4D82-95FE-3EC55DC8FA1F}"
+	ProjectSection(SolutionItems) = preProject
+		..\schemas\JSON\manifests\v1.2.0\manifest.defaultLocale.1.2.0.json = ..\schemas\JSON\manifests\v1.2.0\manifest.defaultLocale.1.2.0.json
+		..\schemas\JSON\manifests\v1.2.0\manifest.installer.1.2.0.json = ..\schemas\JSON\manifests\v1.2.0\manifest.installer.1.2.0.json
+		..\schemas\JSON\manifests\v1.2.0\manifest.locale.1.2.0.json = ..\schemas\JSON\manifests\v1.2.0\manifest.locale.1.2.0.json
+		..\schemas\JSON\manifests\v1.2.0\manifest.singleton.1.2.0.json = ..\schemas\JSON\manifests\v1.2.0\manifest.singleton.1.2.0.json
+		..\schemas\JSON\manifests\v1.2.0\manifest.version.1.2.0.json = ..\schemas\JSON\manifests\v1.2.0\manifest.version.1.2.0.json
+	EndProjectSection
+EndProject

```

Comment:
Is this intentional? Curious why do we list the v1.2 schemas specifically. Also we already have the Project/ManifestSchema that contains organized manifests.

Reply:
Didn't realize they were in the shared projects, removed the whole thing then.